scripts/scr_company_order/scr_company_order.gml

```diff
@@ -2,12 +2,12 @@ function scr_company_order(company) {
 
 	// company : company number
 	// This sorts and crunches the marine variables for the company
-	var co;co=company;
+	var co=company;
 
-	var i, v;i=-1;v=0;
-	var temp_vrace, temp_vloc, temp_vrole, temp_vwep1, temp_vwep2, temp_vup, temp_vhp, temp_vchaos, temp_vpilots, temp_vlid, temp_vwid;
+	var i=-1,v=0;
+	var temp_vrace, temp_vloc, temp_vrole, temp_vwep1, temp_vwep2, temp_vup, temp_vhp, temp_vchaos, temp_vpilots, temp_vlid, temp_vwid, unit;
 
-	for (var i=0;i<401;i++;){i+=1;
+	for (var i=0;i<401;i++;){
```

Comment:
the semicolon after `i++` is probably not needed. I'd remove since I'm not sure how picky is GML about this

Reply:
this is from the gamemaker manual
 for (<assignment>; <expression>; <operation>;){
    <statement>;
    <statement>;
    ...
}

Reply:
the above is the definition of a loop, the actual examples from the manual do not use the 3rd ;

![image](https://github.com/ZAB909/ChapterMaster/assets/63882831/4ffac723-7532-41cf-aea7-8ac10fddcf3a)

Reply:
ok, will remove in which case